pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -171,6 +176,9 @@ func desiredDNSConfigMap(dns *operatorv1.DNS, clusterDomain string, caBundleRevi
 		upstreamResolvers.Policy = dns.Spec.UpstreamResolvers.Policy
 	}
 
+	// Calculate the caching values (in seconds) for use in the Corefile
+	pTTL, nTTL := coreDNSCache(dns)
```

Comment:
nit: can we add something to the function name? `Settings`?

Reply:
If it's not a blocker I'd like to leave it as-is for now and defer the renaming to a later date if more cache settings get surfaced in the API. Mostly because at that point we'll need to discuss whether we want to handle every cache setting in one function or break them out into discrete functions.